docs/sphinx/using/backends/simulators.rst

```diff
@@ -170,9 +170,9 @@ To execute a program on the multi-node multi-GPU NVIDIA target, use the followin
 
     .. note::
         (1) The order of the option settings are interchangeable.
-        For example, `cudaq.set_target('nvidia', option='mgpu,fp64')` is equivalent to `cudaq.set_target('nvidia', option='fp64.mgpu')`.
+        For example, `cudaq.set_target('nvidia', option='mgpu,fp64')` is equivalent to `cudaq.set_target('nvidia', option='fp64,mgpu')`.
 
-        (2) The `nvidia` target has single-precision as the default setting. Thus, using `option='mgpu'` implies that `option='mgpu,fp32'`.  
+        (1) The `nvidia` target has single-precision as the default setting. Thus, using `option='mgpu'` implies that `option='mgpu,fp32'`.  
```

Comment:
This renumbering doesn't appear to be right, screenshot from docs preview:
<img width="721" alt="image" src="https://github.com/user-attachments/assets/390f8588-a02b-4cec-a60e-ddee54018c52">